src/storage/errors.rs

```diff
@@ -1,5 +1,6 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
+//! Types for storage related errors and associated helper methods
```

Comment:
IMO, since the file name is `storge/errors.rs`, it is too obvious to have a single document to describe.

Reply:
I added this after looking at other top level comments present in TiKV/storage - https://tikv.github.io/doc/tikv/storage/index.html (config, types). I can remove it if you feel it's not needed.